Makefile

```diff
@@ -42,7 +42,7 @@ install:
 	chmod 755 ${PREFIX}/bin/Hyprland
 	chmod 755 ${PREFIX}/bin/hyprctl
 	chmod 755 ${PREFIX}/bin/hyprpm
-	ln -s -r -f ${PREFIX}/bin/Hyprland ${PREFIX}/bin/hyprland
+	cd ${PREFIX}/bin && ln -sf Hyprland hyprland
```

Comment:
Do we not need to `cd -` afterwards?

Reply:
Shouldn't be necassary as each command line is executed within a subshell. Otherwise, the style isn't consistent with other `cd` instances in Makefile.

```
$ cd /tmp
$ cat Makefile
all:
	cd /var/empty
	pwd

$ bmake
cd /var/empty
pwd
/tmp

$ gmake
cd /var/empty
pwd
/tmp
```

Reply:
Alright then, thanks for the explanation.

Reply:
One can't reliably `cd` out of an absolute path of arbitrary length. Another option is to wrap the command into parentheses to use current directory of `sh` instead of `make`.